### PR TITLE
test: add initRoleValidation test

### PR DESCRIPTION
**Pull request by GustavoHenriqueRS**
test: InitRoleValidationError

### Testing
Run Automatizated Test, this test is for edit.js 

This PR covers the InitRole Validation test that is present on the book edition page, simulates several test cases that cover all decisions and conditions.

### CODE REVIEW

**Comment by cclauss** [comment]
Please resolve git conflicts.

File: tests/unit/js/editionsEditPage.test.js [hunk 1]
```diff
@@ -208,3 +213,64 @@ describe('initIdentifierValidation', () => {
         expect($('.repeat-item').length).toBe(6);
     });
 });
+
+import { initRoleValidation } from '../../../openlibrary/plugins/openlibrary/js/edit.js';
```
Comment (cdrini) [inline tests/unit/js/editionsEditPage.test.js:217]: Move this to the top of the file.

File: tests/unit/js/editionsEditPage.test.js [hunk 2]
```diff
@@ -20,7 +23,7 @@ describe('isChecksumValidIsbn10', () => {
     it('returns false with an invalid ISBN 10', () => {
         expect(isChecksumValidIsbn10('1234567890')).toBe(false);
     });
-})
+});
```
Comment (cdrini) [inline tests/unit/js/editionsEditPage.test.js:26]: Also undo these formatting changes.

File: tests/unit/js/html-test-data.js [hunk 3]
```diff
@@ -126,7 +126,7 @@ export const clamperSample = `
           <a>orphans</a>
           <a>fantasy fiction</a>
           <a>England in fiction</a>
-      </span>`
+      </span>`;
```
Comment (cdrini) [inline tests/unit/js/html-test-data.js:129]: And these.

**Review by cdrini — CHANGES_REQUESTED** [review]
Thanks @GustavoHenriqueRS ! This looks pretty good, but some automatic formatting changes snuck in which should be removed. Let's please revert those!

**Comment by cdrini** [comment]
Hey @GustavoHenriqueRS ! I'd recommend recreating this PR ; trying to fix those automatic changes will be very difficult. I'd recommend (1) disable the auto-formatting thing you have enabled. Maybe you have prettier?, (2) create a new branch off the latest master, (3) copy your tests from this PR into that fresh branch, and fix any other little fixes. That should make things cleaner!

**Pull request closed without merging**